lineapy/plugins/jinja_templates/airflow_dag_PythonOperator.jinja

```diff
@@ -50,10 +50,16 @@ with DAG(
         python_callable=dag_teardown,
     )
 
-{% for TASK_NAME in tasks %}
-    {{ TASK_NAME }} = PythonOperator(
-        task_id="{{ TASK_NAME }}_task", python_callable=task_{{ TASK_NAME }},
+{% for TASKDESCRIPTION in tasks %}
```

Comment:
if TASKDESCRIPTION is a tuple, we can bring more clarity here by having the var names exploded like so:
`for (TASK_NAME, TASK_PREFIX) in tasks`

Reply:
Second this.

Reply:
done